app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -173,6 +174,9 @@ protected void onCreate(Bundle savedInstanceState) {
         refreshTimer = new EntitySelectRefreshTimer();
         asw = CommCareApplication._().getCurrentSessionWrapper();
         session = asw.getSession();
+        // Don't show actions (e.g. 'register patient', 'claim patient') when
+        // in the middle on workflow triggered by an (sync) action.
+        hideActions = session.isSyncCommand(session.getCommand());

```

Comment:
I'm not sure if this is a good idea, but going to throw it out and we can discuss (or you can reject outright if you hate it)... I agree that it makes sense to reuse the normal case list and case detail display frameworks from the suite file for displaying the case claim options. However, it does feel a little weird to me to be using the exact same code path as the usual entity select screen, with just this boolean flag being the only distinction. You could create a new activity with a more fitting name (`CaseClaimSelectActivity` ?) that extends `EntitySelectActivity`, and then you have a clean way to play around with the UI of that activity a bit and distinguish it from the normal entity select screen. It's a bit of a picky distinction to ask for, but I think it just feels a little off to me to use the same activity to do 2 things that are conceptually very different in the app.